src/lib/ifaces/mac_v.rs

```diff
@@ -8,24 +8,24 @@ use netlink_packet_route::rtnl::nlas::NlasIterator;
 use serde_derive::{Deserialize, Serialize};
 use std::collections::HashMap;
 
-const ETH_ALEN: usize = 6;
+pub const ETH_ALEN: usize = 6;
 
-const MACVLAN_MODE_PRIVATE: u32 = 1;
-const MACVLAN_MODE_VEPA: u32 = 2;
-const MACVLAN_MODE_BRIDGE: u32 = 4;
-const MACVLAN_MODE_PASSTHRU: u32 = 8;
-const MACVLAN_MODE_SOURCE: u32 = 16;
+pub const MACV_MODE_PRIVATE: u32 = 1;
+pub const MACV_MODE_VEPA: u32 = 2;
+pub const MACV_MODE_BRIDGE: u32 = 4;
+pub const MACV_MODE_PASSTHRU: u32 = 8;
+pub const MACV_MODE_SOURCE: u32 = 16;
```

Comment:
These constant is copy from kernel, don't change the name of it. Just add a comment says the `mac_vlan` are sharing the same code with `mac_vtap` in kernel for netlink informations.